StudCity.Frontend/src/components/Sidenav/CreateChatModal/index.jsx

```diff
@@ -0,0 +1,131 @@
+import React, { useState, useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import UploadAvatar from '../../../pages/RegistrationComplete/steps/UploadAvatar';
+import Button from '../../../UI/Button';
+import Avatar from '../../../UI/Avatar';
+import { DEFAULT_AVATAR_URL } from '../../../utils/constants';
+import CustomDialog from '../../../UI/CustomDialog';
+import { createChat } from '../../../app/features/chatsSlice';
+import TextField from '../../../UI/fields/TextField';
+import SearchUsersForCreatedChats from '../../SearchUsersForCreatedChats';
+
+export default function CreateChatModal({ handleClose }) {
+	const [showDialogForAvatar, setShowDialogForAvatar] = useState(false);
+	const [avatar, setAvatar] = useState(DEFAULT_AVATAR_URL);
+	const [nameGroup, setNameGroup] = useState('');
+	const [isDisabledCreateButton, setIsDisabledCreateButton] = useState(false);
+	const [errorMessageWhenButtonDisabled, setErrorMessageWhenButtonDisabled] = useState('');
+	const [idsUsers, setIdsUsers] = useState([]);
+	const [selectedUsers, setSelectedUsers] = useState([]);
+	const dispatch = useDispatch();
+	const handleCloseUploadAvatar = (closed) => {
```

Comment:
from the name of the function I assume that it would only set state showDialogForAvatar to false, if that's the case I think you would better wright function as follows:
const handleCloseAvatarDialog = () => setShowDialogForAvatar(false);